### ISSUE TITLE
Merge psi code into master branch

### ISSUE THREAD

**Issue by mcquinne** [cedardevs/onestop#1025]
### Acceptance Criteria
- [x] psi code merged into onestop according to branching decision from #1024
- [x] **initial merge places all psi code under a subdirectory `psi/`**
    -  avoids widespread initial merge conflicts
    - allows for work on subsequent tasks to proceed in parallel

**NOTE:** Nowadays this can be done with a single command:
`git subtree add --prefix psi git@github.com:cedardevs/psi.git <psi_ref>`
This will place everything from psi as of `<psi_ref>` (e.g. `master` or `v1.3.0`) under the directory `psi/` in the current working tree, which can then be committed to whichever branch we want to add it to.

**Comment by dneufeldcu** [comment]
As per our decision in #1024, the code can be merged directly into master since we will be able to maintain the existing onestop artifacts during the process.

**Comment by mcquinne** [comment]
Resolved by 8c693351
